csp-guru/datapackage.json

```diff
@@ -1,12 +1,12 @@
ved at [Zenodo](https://doi.org/10.5281/zenodo.1318151). \n\n## This version\n The data collection for this version of CSP guru has been conducted by Johan Lilliestam and Richard Thonig. It is based on the May 2018 Version of the csp.guru dataset ( https://doi.org/10.5281/zenodo.1342716 ) by Johan Lilliestam; Mercè Labordena; Lana Ollier. And reflects the situation of the CSP industry on Januart 1st 2019.\n\n## Changes\n The following changes were implemented compared to the last version. \n\n### New Collumns\n We added the following collumns. Support_Currency and Cost_Currency - indicating the currency, i.e. EURO, RMB, USD, etc of both support and Total Cost. For the companies along the supply chain: i.e. Developer,	EPC,	HCE_manufacturer,	SCA_manufacturer we added the respective home countries of the companies to enable the analysis of the localisation of the industry. We reconceptualized the SCA column as to the company engineering the SCA or Heliostat, and not the actual manufacturing, which is often subcontracted to local companies. \n\n### Other Projects \n In terms of projects we decided to limit the dataset to stations that are either under construction, or have been comissioned. Therefore we exclude projects that are part of the NER 300 program , Projects in the Chineese demonstration program that we found no progress information on, and the Solarreserves Redstone project in South Africa and Aurora in Australia that are only. We exlude stations that are under development or have an unclear status. THerefore, we exclude Stillwater and Kogan Creek, MINOS and Noor Midelt. Moreover we exclude the French Alba Nova 1 project which has been canceled, in India there has been no information on the KVK Energy Solar Project, Abhijeet and Diwakar projects for several years, so we view them as canceled. \n In China we previously listed the folowing projects which we could not confirm to be under construction as far as we know: \n* CECIC Gansu Wuwei -100MW Trough, \n* Dahua Shangyi - 50MW Tower, \n* Golden Tower / Gansu Jinta / China Three Gorges New Energy Jinta - 100MW Tower, \n* Golmud - 2 x 100MW Tower, \n*Huaneng Urat Fresnel (Northern United Power Urat) - 50MW Fresnel, \n* Huanghe Delingha - 135MW Tower, \n*Shenzen Jinfan Akesai - 50MW Trough, \n*Shouhang  Yumen (Old name: Shenhua Guohua Yumen) - 100MW Tower, \n*Zhangjiakou Terasolar (Huaqiang Zhaoyang Zhangjiakou DSG 50 CLFR) - 50MW Fresnel, \n* Zhengbei (CITIC Zhenbei) - 50MW Fresnel, \n* Zhongyang Zhangjiakou Chabei - 64MW Trough \n\n### Financial information \n The financial information was reduced compared to the last version. We have not produced LCOE and remuneration for other currencies than the USD and have updated the base year for the LCOE calculation to 2018. For this iteration of the dataset we decided to limit it to materializing projects, that is those under construction and those. Moreover we added the status of decomissioned plants As both SEGS I and II have been decomissioned.",
```

Comment:
Travis has a problem with this line. See https://travis-ci.org/repolicy/csp-guru/builds/590442600?utm_source=github_status&utm_medium=notification

You sure this desciption must be as long?

(unfortunately it didn't fail properly... 🙄)

Reply:
This'll end up in the about section of the website. 
https://github.com/repolicy/csp-guru/blob/website/about.md

Reply:
Travis is failing because this is actually invalid JSON. My editor tells me that "Invalid characters in string. Control characters must be escaped."

1) Use an editor that shows you these error messages (I am using VSCode, Atom should work as well).
2) Check the special characters. I am not quit sure what the problem is.

Reply:
OK, i have pushed everything into a readme, worked on the typos and travis now runs through.